spec/factories/openscap_result.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :openscap_result_for_test, :class => OpenscapResult do
```

Comment:
maybe a slight name change? skip_callback_openscap_result?

(since all factories objects are for tests)

Reply:
I will change to `openscap_result_skip_callback`